### PR TITLE
Calculator components

### PR DESCRIPTION
**Pull request by Samuellekpor**
In this milestone, we:

1. Create the app skeleton with a new component Calculator
2. Add styling to the application

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <button type="button">2</button>
+    <button type="button">3</button>
+    <button type="button" className="orange">+</button>
+    <button type="button" className="double">0</button>
+    <button type="button">.</button>
+    <button type="button" className="orange">=</button>
+  </div>
+);
+
+export default Calculator;
```
Comment (cwaku) [inline src/components/Calculator.js]: - [ ] The `Calculator.js` component should be [class-based](https://betterprogramming.pub/understanding-functional-components-895321b1af84). Kindly convert it from a functional component to a [class-based](https://betterprogramming.pub/understanding-functional-components-895321b1af84) component.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (cwaku) [inline src/App.js]: - [ ] The `App.js` component should be [class-based](https://betterprogramming.pub/understanding-functional-components-895321b1af84). Kindly convert it from a functional component to a [class-based](https://betterprogramming.pub/understanding-functional-components-895321b1af84) component.

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Samuellekpor 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Samuellekpor) [reply src/components/Calculator.js]: Hello @cwaku 
Thanks for the review. I first wrote all of the components class-based but I was getting this error: ''Component should be written as a pure function'' and I didn't know how to solve it. Can you help me, please?

**Review by FrederickMih — COMMENTED** [review]
Hi @Samuellekpor, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive readme.md file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.js (hunk 1 above)
Comment (FrederickMih) [reply src/components/Calculator.js]: Hi @Samuellekpor, have you tried to [extends React.PureComponent](https://www.geeksforgeeks.org/reactjs-pure-components/)?

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Samuellekpor  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**